doc/templates/head.php

```diff
@@ -0,0 +1,7 @@
+<head>
+    <title><?php echo $this->page->getTitle(); ?></title>
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="assets/css/bootstrap-theme.min.css" />
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.6/styles/default.min.css">
```

Comment:
I created a custom bootstrap theme that omitted most features and used or
color scheme, but yes, could definitely do that.
On May 26, 2015 5:55 PM, "Maks3w" <notifications@github.com> wrote:

> In doc/templates/head.php
> <https://github.com/zendframework/zend-diactoros/pull/25#discussion_r31088260>
> :
>
> > @@ -0,0 +1,7 @@
> > +<head>
> > +    <title><?php echo $this->page->getTitle(); ?></title>
> > +    <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
> > +    <link rel="stylesheet" href="assets/css/bootstrap-theme.min.css" />
> > +    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.6/styles/default.min.css">
>
> If external resources loading is expected then bootstrap could be loaded
> from CDN too and only host the customizations
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-diactoros/pull/25/files#r31088260>.
>